packages/cli/src/utils/input.ts

```diff
@@ -238,3 +250,35 @@ export function validateSalt(salt: string, required = false) {
     throw new Error(`Invalid salt ${salt}, must be an uint256 value.`);
   }
 }
+
+export function getPlaceholder(arg: Pick<MethodArg, 'type' | 'components'>): string {
```

Comment:
I don't think "placeholder" is very accurate. What about `getSampleInput`?

Reply:
Like it! The original name was placeholder because the idea was to use it as a placeholder when presenting the question, but it was after that we noted that inquirer does not support placeholders (it only displays them when they are the default value).